velox/exec/Task.cpp

```diff
@@ -2116,10 +2116,10 @@ std::string Task::toShortJsonString() const {
   obj["numThreads"] = numThreads_;
   obj["terminateRequested_"] = std::to_string(terminateRequested_);
   obj["pauseRequested_"] = std::to_string(pauseRequested_);
-  return folly::toPrettyJson(obj);
+  return obj;
 }
 
-std::string Task::toJsonString() const {
+folly::dynamic Task::toDynamic() const {
   std::lock_guard<std::mutex> l(mutex_);
   folly::dynamic obj = folly::dynamic::object;
```

Comment:
Would be nice to call toShortJson() here to initialize the 'obj', then add extra fields.